app/dashboard/models.py

```diff
@@ -4098,7 +4099,7 @@ def get_fulfilled_bounties(self, network=None):
     def get_orgs_bounties(self, network=None):
         network = network or self.get_network()
         url = f"https://github.com/{self.handle}"
-        bounties = Bounty.objects.current().filter(network=network, github_url__istartswith=url).cache()
+        bounties = Bounty.objects.current().filter(network=network, github_url__startswith=url).cache()
```

Comment:
heres the query optimization for above.  it removes the `UPPER()` from the query, which allows the db engine to use the index.